sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientBase.java

```diff
@@ -465,7 +467,7 @@ OnBehalfOfParameters buildOBOFlowParameters(TokenRequestContext request) {
         return builder.build();
     }
 
-    InteractiveRequestParameters.InteractiveRequestParametersBuilder buildInteractiveRequestParameters(TokenRequestContext request, String loginHint, URI redirectUri) {
+    InteractiveRequestParameters.InteractiveRequestParametersBuilder buildInteractiveRequestParameters(TokenRequestContext request, String loginHint, URI redirectUri, BrowserCustomizationOptions browserCustomizationOptions) {
```

Comment:
small nit: `IdentityClientBase` has access to the options, so do we need the parameter here?

Reply:
refactored.